tracer/src/Datadog.Trace/Ci/Agent/MessagePack/CIEventMessagePackFormatter.cs

```diff
@@ -19,7 +19,7 @@ internal class CIEventMessagePackFormatter : EventMessagePackFormatter<EventsPay
         private readonly byte[] _metadataBytes = StringEncoding.UTF8.GetBytes("metadata");
         private readonly byte[] _containerIdBytes = StringEncoding.UTF8.GetBytes("container_id");
         private readonly byte[]? _containerIdValueBytes;
-        private readonly byte[] _runtimeIdBytes = StringEncoding.UTF8.GetBytes("runtime_id");
+        private readonly byte[] _runtimeIdBytes = StringEncoding.UTF8.GetBytes(Trace.Tags.RuntimeId);
```

Comment:
This changes the tag from `"runtime_id"` to `"runtime-id"`, will that cause any issues?

Reply:
It’s part of the fix. I was sending a bad key.